CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing Guidelines
+
+This documentation contains a set of guidelines to help you during the contributing process. We are happy to welcome all the contributors who's willing to enhance/add features to this project. Feel free to contribute and remember, no contribution is too small.
```

Comment:
`all the contributors who are willing to enhance/add documentation and features to this project`